docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -540,8 +535,12 @@ Location named `demo1` with labels `foo=bar` and `the-word=the-bird`.
 This was equivalent to the following commands.
 
 ```shell
-kubectl ws root:imw-1
-kubectl create -f -<<EOF
+imw_space_config="${PWD}/temp-space-config/spaceprovider-default-imw-1"
```

Comment:
The reader is wondering if it is required to include the "temp-space-config/spaceprovider-default-" part. It is NOT required, right? If it is not required then I would omit it here.